cross/libarchive/Makefile

```diff
@@ -0,0 +1,19 @@
+PKG_NAME = libarchive
+PKG_VERS = 3.7.3
+
+PKG_DIST_SITE = https://github.com/libarchive/$(PKG_NAME)/releases/download/v$(PKG_VERS)
+PKG_EXT = tar.xz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+COMMENT = Multi-format archive and compression library
+HOMEPAGE = https://www.libarchive.org/
+LICENSE = 3-Clause BSD
+
+BUILD_DEPENDS = native/autoconf
```

Comment:
It works without native/autoconf.
_and if you do not add the native/autoconf binary folder to the path, the build can't find it_
```suggestion
```